definitions/private-law/json/RoleToAccessProfiles.json

```diff
@@ -1,15 +1,6 @@
 [
   {
-    "RoleName" : "hearing-centre-admin",
-    "Authorisation": "",
-    "CaseTypeID": "PRLAPPS",
-    "AccessProfiles": "caseworker-privatelaw-courtadmin",
-    "ReadOnly": "N",
-    "Disabled": "F",
-    "CaseAccessCategories": ""
-  },
-  {
-    "RoleName" : "hearing-centre-team-leader",
+    "RoleName" : "idam:caseworker-privatelaw-courtadmin",
```

Comment:
Make sure that this file does not go to Prod.This should be reverted before merging to prod.